### PR TITLE
Readability aimec and plot style

### PR DESCRIPTION
**Pull request by fso42**
- Reworked the aimec output some more -> new functions and reordering/deleting
- Plots have a new styling (colormap was not touched)

### CODE REVIEW

**Comment by fso42** [comment]
@matthiasto : there are a few TODOs marked in outAIMEC which I need your feedback on. :smile:

File: avaframe/out3Plot/outAIMEC.py [hunk 1]
```diff
@@ -571,17 +552,43 @@ def resultVisu(cfgPath, rasterTransfo, resAnalysis, plim):
...
     return
+
+
+def _addColorBar(im, ax2, ticks, myUnit):
+    '''
+    Adds, styles and labels a colorbar to the given image and axes
+    '''
+    cbar = ax2.figure.colorbar(im, ax=ax2, ticks=ticks)
+    cbar.outline.set_visible(False)
+    cbar.ax.set_title(myUnit)
```
Comment (matthiasto) [inline avaframe/out3Plot/outAIMEC.py:594]: could we add this in the plot utils instead ?

File: avaframe/out3Plot/outAIMEC.py [hunk 2]
```diff
@@ -571,17 +552,43 @@ def resultVisu(cfgPath, rasterTransfo, resAnalysis, plim):
...
+    if cfgFlags.getboolean('savePlot'):
+        outname = os.path.join(cfgPath['pathResult'], 'pics', outFileName)
+        if not os.path.exists(os.path.dirname(outname)):
+            os.makedirs(os.path.dirname(outname))
+        fig.savefig(outname)
+
+    if cfgFlags.getboolean('plotFigure'):
+        plt.show()
+    else:
+        plt.ioff()
```
Comment (matthiasto) [inline avaframe/out3Plot/outAIMEC.py:581]: could we add this in the plot utils instead ?

File: avaframe/out3Plot/outAIMEC.py (hunk 1 above)
Comment (fso42) [reply avaframe/out3Plot/outAIMEC.py:594]: Ah, good idea!

File: avaframe/out3Plot/outAIMEC.py (hunk 2 above)
Comment (fso42) [reply avaframe/out3Plot/outAIMEC.py:581]: Good idea! Will add this

**Pull request merged**